README.md

```diff
@@ -1,3 +1,21 @@
-# now-go-bridge
+# vercel-go-bridge
```

Comment:
I'd prefer if we rename the repo `vercel/go-bridge` (instead of `vercel/vercel-go-bridge`).

```suggestion
# go-bridge
```